src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/deploy/deploy.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Azion CLI deploy
+description: Implante uma edge application no edge através da Azion CLI.
+meta_tags: cli, next, javascript, astro, vue, react, edge computing
+namespace: documentation_cli_deploy
+menu_namespace: cliMenuAlpha
+permalink: /documentacao/devtools/cli/deploy/
+---
+
+Implante uma edge application no edge através da Azion CLI.
+
+:::note 
+In case one or more required fields aren't informed through the specific flags, an interactive message prompts and asks for the missing information.
```

Comment:
```suggestion
In case one or more required fields aren't informed through the specific flags, an interactive prompt asks for the missing information.
```